src/components/interaction-designer/block-editors/UploadMonitor.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div class="upload-monitor">
+    <div v-if="hasProgress" class="upload-monitor-progress">
+      <h6>{{ 'trees.uploading' | trans }}&hellip; {{ upload.file.name }}</h6>
+
+      <div class="progress">
+        <progress-bar :now="progress.toFixed(1) * 1" label type="default" striped animated/>
+      </div>
+    </div>
+
+    <p v-if="isFailure" class="text-danger small">
+      {{ 'trees.error-uploading-file-try-again'|trans }}
+      <span v-if="upload.message" class="details">({{ upload.message }})</span>
+    </p>
+
+    <slot v-if="!hasProgress"/>
+  </div>
+</template>
+
+<script>
+import lang from 'lib/filters/lang';
+import lodash, { forEach } from 'lodash';
+
+import { mapState } from 'vuex';
+import { progressbar as ProgressBar } from 'vue-strap';
```

Comment:
Should we change this to bootstrap-vue ? 
https://bitbucket.org/voto/voto5/pull-requests/1646/vmo-211-upgrade-to-bootstrap-v45-remove/diff#chg-resources/assets/js/trees/components/block-editors/UploadMonitor.vue

Reply:
or should we add new ticket to upgrade the whole `vue-strap` usage ? eg: https://github.com/FLOIP/flow-builder/blob/master/src/components/interaction-designer/resource-editors/ResourceVariantTextEditor.vue#L95

Reply:
Decision: Try to use boostrap-vue right now if possible

Reply:
@bzabos , @jcbashdown : I realize we're using bootstrap 3 for flow-builder. and `bootstrap-vue` will require bootstrap 4.3. That will break several pages I think.
I suggest we create another ticket to upgrade boostrap from 3 to 4 and change vue-strap usage to boostrap-vue. (similar to Jory's ticket https://bitbucket.org/voto/voto5/pull-requests/1646/vmo-211-upgrade-to-bootstrap-v45-remove/diff)